docs/Deployment/realtime-analytics.md

```diff
@@ -0,0 +1,44 @@
+## Overview
+
+AI DIAL Analytics Realtime is a service that gathers prompt logs from [DIAL Core](https://github.com/epam/ai-dial-core) and publishes them to InfluxDB.
+The logs of prompts are the data a user provides to LLM and the responses the LLM gives to the user. A user can access analytical data through pre-set Grafana dashboards.
+
+**TODO**
+Update AI DIAL Analytics Realtime [README](https://github.com/epam/ai-dial-analytics-realtime/blob/development/README.md) with a link to this document.
+
+## Pre-requisites
+
+- [DIAL Core](https://github.com/epam/ai-dial-core)
+- [Prompt log collector](https://github.com/vectordotdev/vector)
+- [DIAL realtime analytics](https://github.com/epam/ai-dial-analytics-realtime)
+- [Influx DB](https://github.com/influxdata/influxdb)
+- [Grafana](https://github.com/grafana/grafana)
+
+### DIAL Core
+
+Follow the instructions in [AI DIAL Core](https://github.com/epam/ai-dial-core/blob/development/README.md) to setup.
+
+**TODO**
```

Comment:
we can use default config here

Reply:
yes, we can include the default one as a reference.

Reply:
@alexey-ban please attach the link to the config where we have this configured for Core

Reply:
I meant that we can use the configuration already included in the core Docker image. https://github.com/epam/ai-dial-core/blob/development/src/main/resources/gflog.xml